samples/cmake/ozz-animation/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+if(NXDK)
+  # Hack for memory.h
+  file(TOUCH ${CMAKE_CURRENT_BINARY_DIR}/memory.h)
+  include_directories(SYSTEM "${CMAKE_CURRENT_BINARY_DIR}/")
+endif()
+
+set(ozz_build_tools OFF)
+set(ozz_build_fbx OFF)
+set(ozz_build_data OFF)
+set(ozz_build_samples OFF)
+set(ozz_build_howtos OFF)
+set(ozz_build_tests OFF)
+set(ozz_build_simd_ref OFF)
+set(ozz_build_cpp11 OFF)
+set(ozz_build_msvc_rt_dll OFF)
+set(ozz_build_postfix ON)
+add_subdirectory(ozz-animation EXCLUDE_FROM_ALL)
+
+target_compile_options(ozz_base PUBLIC
+  -Dcout=clog
```

Comment:
nxdk-libcxx issue 5: `std::cout` does not exist, but `std::clog` and `std::cerr` exist